File: app/src/main/java/com/worldventures/dreamtrips/social/ui/membership/service/analytics/InviteShareEmailAction.java
package com.worldventures.dreamtrips.social.ui.membership.service.analytics;

import com.worldventures.core.service.analytics.AnalyticsEvent;
import com.worldventures.core.service.analytics.ApptentiveTracker;
import com.worldventures.core.service.analytics.BaseAnalyticsAction;

@AnalyticsEvent(action = "invite_share_send_email",
                trackers = ApptentiveTracker.TRACKER_KEY)
public class InviteShareEmailAction extends BaseAnalyticsAction {

   public InviteShareEmailAction() {
   }
}

File: app/src/main/java/com/worldventures/dreamtrips/social/ui/membership/service/analytics/InviteShareEmailAction.kt
package com.worldventures.dreamtrips.social.ui.membership.service.analytics

import com.worldventures.core.service.analytics.ApptentiveTracker
import com.worldventures.core.service.analytics.BaseAnalyticsAction
import com.worldventures.janet.analytics.AnalyticsEvent

@AnalyticsEvent(action = "invite_share_send_email", trackers = arrayOf(ApptentiveTracker.TRACKER_KEY))
class InviteShareEmailAction : BaseAnalyticsAction()
